.github/workflows/export-dynamic.yaml

```diff
@@ -1,31 +1,22 @@
 name: Export Dynamic Plugin Packages
 on:
-  workflow_dispatch:
+  workflow_call:
     inputs:
       node-version:
         description: node-version to execute the export
+        type: string
         required: false
-        type: choice
-        default: '20.x'
-        options:
-          - '18.x'
-          - '20.x'
+        default: ''
 
       janus-cli-version:
         description: Version of the janus-idp/cli package.
         type: string
         required: false
-        default: '^1.18.0'
-
-      overlay-branch:
-        description: Branch of the overlay structure (current branch by default).
-        type: string
-        required: false
         default: ''
-      
+
       upload-project-on-error:
         description: Upload the complete project as a workflow artifact in case of error in order to troubleshoot.
-        required: false
+        required: true
```

Comment:
Is there a reason you changed this input to `required` ?
It seems strange to have a default value if the input is required.